vtk-cdat/build.sh

```diff
@@ -107,34 +108,53 @@ COMMON_ARGS="-DCMAKE_C_COMPILER=$CC \
         -DVTK_Group_StandAlone:BOOL=OFF \
         -DVTK_LEGACY_SILENT:BOOL=ON"
 
+if [ ${OSNAME} == Linux ]; then
+    LIBEXT="so"
+elif [ ${OSNAME} == Darwin ]; then
+    LIBEXT="dylib"
+fi
 
-if [ `uname` == Linux ]; then
+VTK_ARGS="\
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.12 \
```

Comment:
@sankhesh I think it's merely because on my new laptop I don't have the old SDKs installed so it couldn't build build against older SDKs. `conda-forge` might ask us to swtich to an older SDK anyway.